.goreleaser.yml

```diff
@@ -118,10 +118,6 @@ nfpms:
     builds: ["linux-amd64-musl", "linux-arm64-musl"]
     formats: ["apk"]
 
-furies:
```

Comment:
do we not push to gemfury now?

Reply:
It's marked as invalid by goreleaser, but it's in their documentation...

Wha?

Reply:
I'll reinstate it and raise another issue.